src/utils/lazy.ts

```diff
@@ -7,13 +7,17 @@ export class Lazy<T> {
     private _isValueCreated: boolean = false;
     private _value: T | undefined;
 
-    public constructor(private readonly valueFactory: () => T, private readonly _valueLifetime?: number) {
+    public constructor(private readonly valueFactory: () => T, private _valueLifetime?: number) {
     }
 
     public get isValueCreated(): boolean {
         return this._isValueCreated;
     }
 
+    public clearLifeTime(): void {
```

Comment:
(minor name change suggestion--represents intent more clearly)
 
```suggestion
    public cacheForever(): void {
```